### PR TITLE
HL-72

### PR DESCRIPTION
**Pull request by xiangzhe0812**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-72)

update deployment script.

### CODE REVIEW

**Review by brettvniekerk — APPROVED** [review]

File: .github/workflows/deploy-uat.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    branches:
+      - development
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 16.x
```
Comment (xcjlo985) [inline .github/workflows/deploy-uat.yml:14]: Pull the node version from package.json as per GHAs in other recent projects

File: .github/workflows/deploy-uat.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - name: deploy
+        uses: appleboy/ssh-action@v0.1.4
+        with:
+          host: "${{ secrets.SSH_HOST }}"
+          username: "${{ secrets.SSH_USERNAME }}"
+          key: "${{ secrets.SSH_KEY }}"
+          script: |
+            cd ~/api.horse.link &&
```
Comment (xcjlo985) [inline .github/workflows/deploy-uat.yml:23]: Why `&&` for each cmd? Seems redundant

**Review by xcjlo985 — COMMENTED** [review]
Please add workflow_dispatch. That way you can execute the action manually - and that includes on this pr. Example [here](https://github.com/dltxio/react-template/blob/main/.github/workflows/test.yml)

**Review by xcjlo985 — APPROVED** [review]
Not sure why but I cannot seem to access the action on the branch. Will have to test after merge I guess

**Review by OptimusOpus — APPROVED** [review]

**Pull request merged**